### PR TITLE
MINOR: Add Xin Zhang to committers page

### PR DESCRIPTION
**Pull request by coderex2522**
### What changes were proposed in this pull request?
This PR aims to add Xin Zhang  to the committers page.

### Why are the changes needed?
To keep the list of committers for the ORC website up to date.

### How was this patch tested?
Manually review.

### CODE REVIEW

File: site/develop/committers.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Vaibhav Gumashta        | vgumashta    | Committer
 Wei Zheng               | weiz         | Committer
 William Hyun            | william      | PMC Chair
 Xiening Dai             | xndai        | Committer
+Xin Zhang               | xinzhang     | Committer
```
Comment (dongjoon-hyun) [inline site/develop/committers.md:54]: Thank you. Please sort by Apache ID as mentioned here, @coderex2522 .
https://github.com/apache/orc/blob/e674d8a61d77d0e477c5251f5091b07e1c3c1d3e/site/develop/committers.md#L9

**Review by dongjoon-hyun — COMMENTED** [review]
Also, cc @williamhyun and @wgtmac , too.

**Review by wgtmac — APPROVED** [review]

**Comment by wgtmac** [comment]
@dongjoon-hyun Should I add a welcome page as well?

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by dongjoon-hyun** [comment]
Do you mean `News`, @wgtmac ?
> @dongjoon-hyun Should I add a welcome page as well?

**Review by williamhyun — APPROVED** [review]
+1 LGTM

**Comment by williamhyun** [comment]
@wgtmac 
Yes, please add an entry to `News` welcoming Xin as a new committer!

**Pull request merged**